.changeset/selfish-books-fail.md

```diff
@@ -0,0 +1,8 @@
+---
+"@comet/admin-date-time": major
+"@comet/admin": major
+---
+
+All `FinalFormField` components now support the `required` prop
+
+The `clearable` prop is set per default in `FinalFormDatePicker` when `required` is not set
```

Comment:
> `clearable` prop is set per default

This isn't really correct anymore, as the prop has been removed.

Should this maybe be two separate changesets? 
- The "fix" for the `required` prop, that is now passed to all components
- The removal of the `clearable` prop and the clear-button being dependent on `required`